server/controllers/stock/inventory.js

```diff
@@ -42,11 +42,14 @@ var core        = require('./inventory/core'),
     types       = require('./inventory/types'),
     units       = require('./inventory/units');
 
+var listReceipt = require('./inventory/receipts/list');
+
 // exposed routes
-exports.createInventoryItems  = createInventoryItems;
-exports.updateInventoryItems  = updateInventoryItems;
-exports.getInventoryItems     = getInventoryItems;
-exports.getInventoryItemsById = getInventoryItemsById;
+exports.createInventoryItems    = createInventoryItems;
```

Comment:
Just leaving a note - eventually, we should migrate the inventory to using the regular bhima [2.x guidelines](https://github.com/IMA-WorldHealth/bhima-2.X/wiki/Testing:-Integration-Testing-Server-APIs).